.github/ISSUE_TEMPLATE/russian.yaml

```diff
@@ -0,0 +1,51 @@
+name: Сообщить о проблеме
```

Comment:
we can remove the ru.md and also lets name ths one ru.yaml.

sposiba ! :)

Reply:
Done!

You're welcome! :)